libraries/AP_GPS/AP_GPS_SBF.cpp

```diff
@@ -47,7 +47,7 @@ AP_GPS_SBF::AP_GPS_SBF(AP_GPS &_gps, AP_GPS::GPS_State &_state,
 {
     sbf_msg.sbf_state = sbf_msg_parser_t::PREAMBLE1;
 
-    port->write((const uint8_t*)_initialisation_blob[0], strlen(_initialisation_blob[0]));
+    port->write((const uint8_t*)_port_enable, strlen(_port_enable));
```

Comment:
Shouldn't you set *_init_blob_advance_time* here? Also, either I'm failing to understand what it does or it seems a bad name, I don't get how it relates to what it is used for (I would say something like *_port_enable_time* would be better).

Reply:
You're right, it should be set here. It was meant called advance time as if we have received a message and thus advanced which message we were sending we didn't need to send a port enable. `_port_enable_time` would work, or it could almost be `_config_last_ack_time` which might be more clear?

Reply:
I like `_config_last_ack_time`, it's not 100% accurate, but very close to it, so I think it's even better than `_port_enable_time`.